appengine/cloudsql-mysql/Gemfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Google, Inc
```

Comment:
I grouped both mysql and postgres additions as "transitioning for Cloud SQL PostgreSQL". SGTM!